uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/ICustomBrokerInstallationTest.java

```diff
@@ -0,0 +1,66 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.ui.automation;
+
+/**
+ * An interface to facilitate testing a very specific broker installation order with support for
+ * old and current version of the broker.
+ */
+public interface ICustomBrokerInstallationTest {
+
+    /**
+     * Install old/legacy BrokerHost
+     */
+    void installOldBrokerHost();
+    /**
+     * Install updated BrokerHost
+     */
+    void installBrokerHost();
```

Comment:
is this going to replace the update() method in App.java? 
calling installOldBrokerHost() and installBrokerHost() is going to update the broker host right?

Reply:
Followed up online, Sowmya to investigate if the -r flad in updte() is significant.

Reply:
If these methods here i.e. `installBrokerHost` , `installAuthenticator` etc. are updating the broker then I suggest we rename these accordingly.

Reply:
They don't update it per se. More so they install the latest apk, regardless of whether or not an existing one exists. The question is whether or not this is actually the same as updating the app.

Reply:
@fadidurah adb install -r test.apk reinstalls the app, keeping its data.
So the question here is, do we want to install OldBrokerHost and then the new one by keeping the data? We need to confirm this with Dome on whether we want to retain the data. If yes, we should ideally update the app with -r instead of just installing new one on top of the old one.

Reply:
In that case, I think that this approach will be ideal, because we will have access to both a new installation and an update method. Perhaps we can outline which to use as part of the test case itself?